tests/e2e/utils/product-editor.js

```diff
@@ -577,17 +577,15 @@ export function getProductBlockEditorUtils( page ) {
 		async assertUnableSave( message = 'Please enter a valid value.' ) {
 			await this.clickSave();
 
-			const failureNotice = page
-				.locator( '.components-snackbar__content' )
-				.filter( { hasText: new RegExp( message ) } );
-
-			const failureNoticeDismissButton =
-				failureNotice.getByRole( 'button' );
+			const failureNotice = page.locator(
+				'.components-snackbar__content'
+			);
 
 			await expect( failureNotice ).toBeVisible();
+			await expect( failureNotice ).toHaveText( message );
```

Comment:
This assertion failed with WC < 9.2. How about making it also compatible?

```suggestion
			await expect( failureNotice ).toContainText( message );
```

Reply:
Thanks for the suggestion I've updated it here: [51e3679](https://github.com/woocommerce/google-listings-and-ads/pull/2511/commits/51e36797ff3f70569e16539409aed4990bd5120d)